### PR TITLE
Basic TensorFlow integration

### PR DESCRIPTION
**Pull request by bratseth**
This wil replace any occurrence of tensorflow(...)
in ranking expressions with the corresponding translated expression.

It is functional but these tings are outstanding
- Propagate warnings
- Import a model just once even if referred multiple times
- Add constants as tensor files rather than config

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+            // Find the specified expression
+            ImportResult result = tensorFlowImporter.importModel(asString(feature.getArguments().expressions().get(0)));
+            ImportResult.Signature signature = chooseOrDefault("signatures", result.signatures(),
+                                                               optionalArgument(1, feature.getArguments()));
+            String output = chooseOrDefault("outputs", signature.outputs(),
+                                            optionalArgument(2, feature.getArguments()));
+
+            // Add all constants
+            result.constants().forEach((k, v) -> profile.addConstantTensor(k, new TensorValue(v)));
```
Comment (lesters) [inline config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java]: How does this work for potentially large tensors?

File: config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+            return result.expressions().get(output).getRoot();
+        }
```
Comment (lesters) [inline config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java]: We could consider validating that macros for the placeholders actually are present. Or issue a warning if not found?

**Review by lesters — APPROVED** [review]
LGTM. Please consider comments.

File: config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java (hunk 2 above)
Comment (bratseth) [reply config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java]: Yes, but I think we already do validate that during deployment.

File: config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java (hunk 1 above)
Comment (bratseth) [reply config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/TensorFlowFeatureConverter.java]: Not at all. See outstanding work.

**Pull request merged**